src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -2704,92 +2694,3 @@ export const handleUpdateTwilio = [
   validateTwilioCredentials,
   updateTwilioCredentials,
 ] as ControllerHandler[]
-
-/**
- * NOTE: Exported for testing.
- * Private handler for PUT /forms/:formId/payment
- * @precondition Must be preceded by request validation
- * @security session
- *
- * @returns 200 with updated payments
- * @returns 403 when current user does not have permissions to update the payments
- * @returns 404 when form cannot be found
- * @returns 410 when updating the payments for an archived form
- * @returns 422 when user in session cannot be retrieved from the database
- * @returns 500 when database error occurs
- */
-export const _handleUpdatePayments: ControllerHandler<
-  { formId: string },
-  IEncryptedFormDocument['payments_field'] | ErrorDto,
```

Comment:
These functions moved to `admin-form.payments.controller` in tandem with moving payments routes to `admin-form.payments.routes`.